ogr/ogrsf_frmts/miramon/ogrmiramondriver.cpp

```diff
@@ -0,0 +1,249 @@
+/******************************************************************************
+ *
+ * Project:  OpenGIS Simple Features Reference Implementation
+ * Purpose:  Implements OGRMiraMonDriver class.
+ * Author:   Abel Pau
+ ******************************************************************************
+ * Copyright (c) 2024, Xavier Pons
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE. 
+ ****************************************************************************/
+
+#include "ogrmiramon.h"
+
+/****************************************************************************/
+/*                    OGRMMDriverIdentify()                                 */
+/****************************************************************************/
+
+static int OGRMiraMonDriverIdentify(GDALOpenInfo *poOpenInfo)
+
+{
+    if (poOpenInfo->fpL == nullptr || poOpenInfo->nHeaderBytes < 7)
+        return FALSE;
+    else if (EQUAL(CPLGetExtension(poOpenInfo->pszFilename), "PNT") ||
+             EQUAL(CPLGetExtension(poOpenInfo->pszFilename), "ARC") ||
+             EQUAL(CPLGetExtension(poOpenInfo->pszFilename), "POL"))
+    {
+        if (!poOpenInfo->TryToIngest(7))
+            return FALSE;
+
```

Comment:
unneeded. GDALOpenInfo tries to read at least 1024 bytes (or up to the file size if less than 1024), so if there are at least 7 bytes, they will have been read already

```suggestion
```

Reply:
Ok!